common/source/docs/common-daiwa-winch.rst

```diff
@@ -66,8 +66,23 @@ Set the following parameters
 - :ref:`RC6_TRIM <RC6_TRIM>` = The mid value between :ref:`RC6_MIN <RC6_MIN>` and :ref:`RC6_MAX <RC6_MAX>` which is normally close to 1500
 - :ref:`RC8_OPTION <RC8_OPTION>` = 44 (Winch Enable) to allow relaxing the winch by pulling the transmitter's channel 8 switch low
 
+Winch Options
+=============
+
+The :ref:`WINCH_OPTIONS<WINCH_OPTIONS>` parameter provides a bit mask of possible configuration options for the winch when the following bits are set in the parameter:
+
+===     ======
+Bit     Option
+===     ======
+0       Spin freely on startup allows the line to be pulled out after startup. This is the current default but is actually quite annoying because we need to remember to disengage the clutch before takeoff or the package will likely be left on the ground.
```

Comment:
We can shorten this one to just, "Spin freely on startup allows the line to be pulled out after startup"

Reply:
will do